### ISSUE TITLE
Audio UI components

### ISSUE THREAD

**Issue by benbovy** [benbovy/ipytone#11]
[NexusUI](https://nexus-js.github.io/ui/) provides a lot of great UI components that we could wrap into DOM widgets.

**Comment by davidbrochart** [comment]
There is also [GridSound](https://gridsound.com/daw/): https://github.com/gridsound/daw

**Comment by benbovy** [comment]
Let's turn Jupyter into a full-featured DAW :-)

**Comment by davidbrochart** [comment]
:smile: GridSound is more like a monolith though, I'm not sure it plays well with ipywidgets' layout system for instance. And it looks like it was not written to be used by others.

**Comment by benbovy** [comment]
Yes it will probably be easier to first experiment with NexusUI elements. SoundGrid looks really impressive, though!

**Comment by benbovy** [comment]
[react-nexusui](https://github.com/rakannimer/react-nexusui) will probably be easier to integrate in ipytone?

Some more widgets:

- [Oscilloscope](https://www.npmjs.com/package/oscilloscope)
- [react-audio-spectrum](https://www.npmjs.com/package/react-audio-spectrum)

**Comment by benbovy** [comment]
https://github.com/g200kg/webaudio-controls
